ext/sockets/sockets.c

```diff
@@ -1719,7 +1719,8 @@ PHP_FUNCTION(socket_sendto)
 		case AF_UNIX:
 			memset(&s_un, 0, sizeof(s_un));
 			s_un.sun_family = AF_UNIX;
-			snprintf(s_un.sun_path, 108, "%s", addr);
+			/* macOS and FreeBSD sun_path is 104 bytes instead of 108 on Linux */
+			snprintf(s_un.sun_path, 104, "%s", addr);
```

Comment:
Should this possibly be `sizeof(s_un.sun_path)` instead?

Reply:
Makes sense will change this.